bin/autoproj_bootstrap

```diff
@@ -601,4 +601,4 @@ if !ops.skip_stage2?
     ops.call_stage2
 end
 ops.run_autoproj 'bootstrap', *bootstrap_options
```

Comment:
Ok. So raise in `#run_autoproj` and rescue somewhere? Btw, I would have to modify both autoproj_bootstrap and ops/install.rb, right?